x-pack/plugins/fleet/server/services/agents/crud.ts

```diff
@@ -289,6 +294,7 @@ export async function getAgentsByKuery(
           }),
       ...(pitId && searchAfter ? { search_after: searchAfter, from: 0 } : {}),
       ...(getStatusSummary && { aggs: { status: { terms: { field: 'status' } } } }),
+      ...(aggregations && { aggs: aggregations }),
```

Comment:
please make sure that aggs are going when we pass `getStatusSummary` and `aggregations` params